examples/get-started/pure-js-google-maps/app.js

```diff
@@ -0,0 +1,49 @@
+/* global document, google */
+import {GoogleMapsOverlay as DeckOverlay} from '@deck.gl/google-maps';
+import {GeoJsonLayer} from '@deck.gl/layers';
+
+// source: Natural Earth http://www.naturalearthdata.com/ via geojson.xyz
+const GEOJSON =
+  'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_110m_admin_1_states_provinces_shp.geojson'; //eslint-disable-line
+
+// Retrieving GOOGLE_MAPS_API_KEY from the environment variable
+const gmUrl = `https://maps.googleapis.com/maps/api/js?key=${
+  process.env.GOOGLE_MAPS_API_KEY // eslint-disable-line
+}&libraries=visualization&v=3.34`;
+
+function loadScript(url) {
+  const script = document.createElement('script');
+  script.type = 'text/javascript';
+  script.src = url;
+  const head = document.querySelector('head');
+  head.appendChild(script);
+  return new Promise(resolve => {
+    script.onload = resolve;
+  });
+}
+
+loadScript(gmUrl).then(() => {
+  const map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 40, lng: -100},
+    zoom: 4
+  });
+
+  const overlay = new DeckOverlay({
```

Comment:
Just a thought: The example looks somewhat bland. Can't really put my finger on it. The way the overlay color mixes with the underlying map perhaps (the colors of the google map are a bit vibrant and competes visually with the overlay...)

It would just be nice to to show something that "pops" a little more.

Reply:
If you don't mind adding an extra JSON file to this example, there are some more subtle basemaps on https://snazzymaps.com/. Usage:

```js
const styles = await fetch('styles.json').then((r) => r.json());
const map = new google.maps.Map(el, {styles, ...});
```

Reply:
We can definitely change it to something more interesting. Though I would like all get-started examples to use the same dataset/layer. I will make another pass in a follow up PR.